spec/support/elasticsearch_helpers.rb

```diff
@@ -10,7 +10,7 @@ def sync_elasticsearch(crops)
 RSpec.configure do |config|
   config.include ElasticsearchHelpers
 
-  config.before(:each) do
+  config.before do
```

Comment:
I don't think there's any benefit to replacing `before(:each)` with `before` here - though maybe we could make it a `before(:all)` block instead of a `before(:each)` block?